plugins/schedule/base/lib/model/LiveStreamScheduleEvent.php

```diff
@@ -31,4 +65,33 @@ public function applyDefaultValues()
 		parent::applyDefaultValues();
 		$this->setType(ScheduleEventType::LIVE_STREAM);
 	}
+
+	public function postInsert(PropelPDO $con = null)
+	{
+		parent::postInsert($con);
+		$this->addCapabilityToTemplateEntry($con);

```

Comment:
Why do we need to add it in the post update/insert? cant we set this value when building the object before save?

Reply:
Because we update the source entry if exist - different object on the system that in this object we just keep its ID